src/views/Alteration.vue

```diff
@@ -215,6 +215,30 @@ export default class Alteration extends Mixins(CommonMixin, FeeMixin, FilingTemp
     return null
   }
 
+  @Watch('hasBusinessTypeChanged')
+  onBusinessTypeChanged () {
+    const filingData = this.getFilingData
+    // When we are converting from BC to ULC, it's $1000 not $100.
+    if (this.getEntitySnapshot?.businessInfo?.legalType === CorpTypeCd.BC_COMPANY &&
+        this.getEntityType === CorpTypeCd.BC_ULC_COMPANY) {
+      // Travis Semple - will use ENUM after bcrs-shared-components has been updated.
+      this.setFilingData([{
+        filingTypeCode: 'NOALU' as FilingCodes,
```

Comment:
Is there an easy way to use different filing data (in a resource file?) so we don't need a special case here?